core-java-modules/core-java-arrays-multidimensional/src/test/java/com/baeldung/array/reversearray/ReverseArrayElementsTest.java

```diff
@@ -0,0 +1,79 @@
+package com.baeldung.array.reversearray;
+
+import static com.baeldung.array.reversearray.ReverseArrayElements.toReversedList;
+import static java.util.Arrays.asList;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+
+class ReverseArrayElementsTest {
+
+    @Test
+    void givenArray_whenCallReverseRows_thenExpectAllRowsReversed() {
+        int[][] input = new int[][] { { 1, 2, 3 }, { 3, 2, 1 }, { 2, 1, 3 } };
+
+        int[][] expected = new int[][] { { 3, 2, 1 }, { 1, 2, 3 }, { 3, 1, 2 } };
+        ReverseArrayElements.reverseRowsUsingSimpleForLoops(input);
+
+        assertThat(input, is(expected));
```

Comment:
I find assertJ simpler and more meaningful.
```suggestion
        assertThat(input).isEqualTo(expected);
```